bundles/io/org.openhab.io.myopenhab/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,54 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.io.myopenhab.internal
+Ignore-Package: org.openhab.io.myopenhab.internal
+Bundle-ManifestVersion: 2
+Bundle-Name: my.openHAB Service Connection Bundle
+Bundle-SymbolicName: org.openhab.io.myopenhab
+Bundle-Version: 1.7.0.qualifier
+Bundle-Activator: org.openhab.io.myopenhab.internal.MyOpenHABServiceActivator
+Service-Component: OSGI-INF/myopenhab.xml
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Import-Package: org.apache.commons.io;version="2.0.1",
+ org.apache.commons.lang;version="2.6.0",
```

Comment:
so you are using 2.6 already rather 3.x – please remove version unless you really require on this version